ethereum/contracts/Governance.sol

```diff
@@ -10,13 +10,21 @@ import "./Setters.sol";
 
 import "@openzeppelin/contracts/proxy/ERC1967/ERC1967Upgrade.sol";
 
+interface CoreBridgeModule {
+  function getModule() external returns (bytes32);
+}
+
 abstract contract Governance is GovernanceStructs, Messages, Setters, ERC1967Upgrade {
     event ContractUpgraded(address indexed oldContract, address indexed newContract);
     event GuardianSetAdded(uint32 indexed index);
 
     // "Core" (left padded)
     bytes32 constant module = 0x00000000000000000000000000000000000000000000000000000000436f7265;
 
+    function getModule() public pure returns (bytes32) {
```

Comment:
It looks like none of these contracts actually _implement_ the associated interface (in the sense of nominal typing, `... is CoreBridgeModule`). Is that intentional?